core-concepts/plugins.md

```diff
@@ -1,200 +1,201 @@
 
-## Augmenting a Store
+## Aumentar un Almacén {#augmenting-a-store}
 
-You can add properties to every store by simply returning an object of them in a plugin:
+Puedes añadir propiedades a cualquier almacén solo retornando un objecto de estos mismo en un plugin:
 
 ```js
 pinia.use(() => ({ hello: 'world' }))
 ```
 
-You can also set the property directly on the `store` but **if possible use the return version so they can be automatically tracked by devtools**:
+También puedes colocar la propiedad directamente en la `store` pero **si es posible usa la versión retornada para que pueda ser seguida automáticamente por las herramientas de desarrollo**:
 
 ```js
 pinia.use(({ store }) => {
   store.hello = 'world'
 })
 ```
 
-Any property _returned_ by a plugin will be automatically tracked by devtools so in order to make `hello` visible in devtools, make sure to add it to `store._customProperties` **in dev mode only** if you want to debug it in devtools:
+Cualquier propiedad _retornada_ por un plugin será automáticamente seguido por las herramientas de desarrollo para hacer que `hello` sea visible en las herramientas de desarrollo. Asegúrate de añadirlo a `store._customProperties` **solo en modo de desarrollo** si quieres depurarlo en las herramientas de desarrollo:
 
 ```js
-// from the example above
+// del ejemplo anterior
 pinia.use(({ store }) => {
   store.hello = 'world'
-  // make sure your bundler handle this. webpack and vite should do it by default
+  // asegúrate que tu bundler maneje esto. webpack y vite deberían hacerlo por defecto
   if (process.env.NODE_ENV === 'development') {
-    // add any keys you set on the store
+    // añade cualquier clave que crees al almacén
     store._customProperties.add('hello')
   }
 })
 ```
 
-Note that every store is wrapped with [`reactive`](https://v3.vuejs.org/api/basic-reactivity.html#reactive), automatically unwrapping any Ref (`ref()`, `computed()`, ...) it contains:
+Cabe aclarar que cualquier almacén está envuelto con [`reactive`](https://v3.vuejs.org/api/basic-reactivity.html#reactive), por lo que desenvuelve automáticamente cualquier Ref (`ref()`, `computed()`, ...) que este contiene:
 
 ```js
 const sharedRef = ref('shared')
 pinia.use(({ store }) => {
-  // each store has its individual `hello` property
+  // cada almacén tiene sus propias propiedades `hello`
   store.hello = ref('secret')
-  // it gets automatically unwrapped
+  // se desenvuelven automáticamente
   store.hello // 'secret'
 
-  // all stores are sharing the value `shared` property
+  // todos los almacenes comparten el valor de la propiedad `shared`
   store.shared = sharedRef
   store.shared // 'shared'
 })
 ```
 
-This is why you can access all computed properties without `.value` and why they are reactive.
+Esto es el porque puedes acceder a todas las propiedades computadas sin `.value` y porque son reactivas.
```

Comment:
I think that we can improve to "Por eso se puede acceder a todas las propiedades computadas sin `.value` y por eso son reactivas." What do you think?

Reply:
you're right